README.md

```diff
@@ -78,12 +78,23 @@ A _Pre-SBE_ repo is a repo that is built before source-build-externals during th
 
 When updating a component that is used in a Pre-SBE repo, please adhere to the following steps:
 
-1. **Add the updated component**: Include the component with the updated version as a new submodule in source-build-externals named `<component>-<new-version>` (this new submodule should be separate from any previously existing version's submodule). Rename the component's old submodule to `<component>-<old-version>`. Also rename the old component's patches directory in `patches/` (if it exists) and the project file in `repo-projects/` to match the submodule name of `<component>-<old-version>`.
-    - See [#276](https://github.com/dotnet/source-build-externals/pull/276) for an example.
-2. **Update post-SBE repositories (if applicable)**: If your dependency is also in a repo that is built after SBE, allow the updated version to flow to the post-SBE repository and update the post-SBE repository to use this new version.
-3. **Build and release**: Perform a build and release the artifacts to update the n-1 (previous) artifacts for the pre-SBE repository.
-4. **Update the pre-SBE repository**: Update the pre-SBE repository to use the new version of the component.
-5. **Clean up**: Remove the outdated component from source-build-externals, the outdated component's patches directory in `patches/`, if applicable, and the project file in `repo-projects`. Rename the updated component's submodule to `<component>`. Also rename any existing patches directory in `patches/` and the respective project file in `repo-projects/` to match the submodule name, `<component>`.
+1. **Include the component in Versions.props and Version.Details.xml**:
+
+    1. **Locate the Pre-SBE repository**: This is where you'll be making changes.
+
+    1. **Check `eng/Versions.props`**: Look for a version property for your component. If it doesn't exist, you'll need to add it.
+
+    1. **Check `eng/Version.Details.xml`**: Similarly, look for an entry for your component. If it doesn't exist, add it.
+
+    1.  **Add a descriptive comment**: When adding an entry to `Version.Details.xml`, include a comment above the entry that describes what it is.
+
+    1. **Examples**: For reference, you can check these examples of [`eng/Versions.props`](https://github.com/dotnet/arcade/pull/14698/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67R81) and [`eng/Version.Details.xml`](https://github.com/dotnet/arcade/pull/14698/files#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3R8).
+
+1. **Update the component**: 
+
+    1. **Wait for changes to propagate**: After making changes to the Versions.props and Version.Details.xml, you need to wait for these changes to flow to the [VMR](https://github.com/dotnet/dotnet).
```

Comment:
There is no need to wait with this pattern if the N-1 version is acceptable, if not you must wait for a release in servicing or request a rebootstrap in previews.

Reply:
Won't the build break if the SBE change flows in to the VMR before the version changes from the Pre-SBE repo flow in to the VMR?

Reply:
Source-build will lift/lower the version via the package.versions.props mechanism.  The only way this would break is if there is an api break or functional change that is required.

Reply:
Doesn't the PVP mechanism depend on the entry being in the Version.Details? In this step, I'm trying to convey that if an entry for the component was added to the Version.Details.xml file, the user would have to wait for that change to the Version.Details.xml to flow into the VMR before updating the component in SBE. Otherwise, the SBE component change could flow in to the VMR before the Version.Details change and cause a pre-built.

Reply:
I see. I think the wording could be improved to call out when exactly waiting is needed.  If the Version.Details.xml dependency and corresponding property in the versions.props, then there is no need to wait at the infra is already in place for SB to modify the version.

Reply:
Updated the wording in https://github.com/dotnet/source-build-externals/pull/304/commits/6bd4456956930d0bc7216ad6583d17997edf9866.